docs/install.rst

```diff
@@ -6,6 +6,19 @@ From Binary
 
 To download pre-built binaries, see the `Download page <https://tendermint.com/download>`__.
 
+From Source using Docker
+------------------------
+
+If you have docker running, all you need is the ``golang`` image to build tendermint.
+If you don't, you can get help setting it up `here <https://docs.docker.com/engine/installation/>`__.
+
+::
+    mkdir $HOME/tendermintbin
+    docker run --rm -it -v $HOME/tendermintbin:/go/bin:Z golang:1.9.2 /bin/bash -c "go-wrapper download github.com/tendermint/tendermint/cmd/tendermint ; make -C /go/src/github.com/tendermint/tendermint get_tools get_vendor_deps build_cc"
```

Comment:
No idea what go-wrapper does.

This isn't very helpful because it's annoying to download all the dependencies every time.

I think it would be much better to show devdoc usage and explain how it works.  We probably don't have to explain to anyone how to run build_cc... cross-compiling is only for us to use for CI or releasing, no?

Reply:
```
> git checkout -b feature/devdoc origin/feature/devdoc // TODO merge to develop
> git reset --hard origin/feature/devdoc // TODO merge to develop
> make devdoc_init
> make devdoc
% make get_tools
% make get_vendor_deps
% make build
...
// Exit and checkpoint as a new layer.
% exit
> make devdoc_save

// Restart from the last checkpoint.
> make devdoc
```

Reply:
The remaining instructions refer to `go get github.com/tendermint/tendermint/cmd/tendermint`, which is not the right way to build from source.